File: SDPCheckers/GameClasses/GameStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SDPCheckers.GameClasses
{
    class GameStatus
    {

        private string _status;
        public int status {
            get {
                return Convert.ToInt32(_status);
            }
            set {
                _status = value.ToString();
            }
        }

        private string _gameID;
        public int gameID {
            get
            {
                return Convert.ToInt32(_gameID);
            }
            set
            {
                _gameID = value.ToString();
            }
        }

        private string _numOfPlayers;
        public int numOfPlayers {
            get
            {
                return Convert.ToInt32(_numOfPlayers);
            }
            set
            {
                _numOfPlayers = value.ToString();
            }
        }

        private string _currentPlayerTurn;
        public int currentPlayerTurn {
            get
            {
                return Convert.ToInt32(_currentPlayerTurn);
            }
            set
            {
                _currentPlayerTurn = value.ToString();
            }
        }

        public string _sourceCol;
        public int sourceCol {
            get
            {
                return Convert.ToInt32(_sourceCol);
            }
            set
            {
                _sourceCol = value.ToString();
            }
        }

        public string _sourceRow;
        public int sourceRow {
            get
            {
                return Convert.ToInt32(_sourceRow);
            }
            set
            {
                _sourceRow = value.ToString();
            }
        }

        public string _destCol;
        public int destCol {
            get
            {
                return Convert.ToInt32(_destCol);
            }
            set
            {
                _destCol = value.ToString();
            }
        }

        public string _destRow;
        public int destRow {
            get
            {
                return Convert.ToInt32(_destRow);
            }
            set
            {
                _destRow = value.ToString();
            }
        }

        public GameStatus()
        {

        }
    }
}

File: GameStatus.php
<?php

	// Prepare variables for database connection
	
	$dbusername = "AG968";  // enter database username
	$dbpassword = "mahmoudag";  // enter database password
	$server = "www.abugharbieh.com"; 
	$dbname = "SeniorDesignProject";
	
	// Create connection
	$conn = new mysqli($server, $dbusername, $dbpassword, $dbname);
	
	// Check connection
	if ($conn->connect_error) {
	    echo "CONNECTION FAILED";
	    die("Connection failed: " . $conn->connect_error);
	} 


      //$dbconnect = mysql_pconnect($server, $dbusername, $dbpassword);
      //$dbselect = mysql_select_db("SeniorDesignProject",$dbconnect);

	//POST request handler
	if ($_SERVER['REQUEST_METHOD'] === 'POST') 
	{
	
		//Make sure all required POST parameters have been passed
		if( isset(
			$_POST['gameID'], 
			$_POST['numOfPlayers'], 
			$_POST['currentPlayerTurn'],
			$_POST['sourceCol'],
			$_POST['sourceRow'],
			$_POST['destCol'],
		$_POST['destRow']
		))
		{
			//$sql = "SELECT * FROM  `Games` WHERE  `gameID` = 1";
			//Update the DB of the game with the proper POST parameters
			$sql = 
			"UPDATE 
				Games 
			SET 
				numOfPlayers = '".$_POST['numOfPlayers']."',
				currentPlayerTurn = '".$_POST['currentPlayerTurn']."',
				sourceCol = '".$_POST['sourceCol']."', 
				sourceRow = '".$_POST['sourceRow']."',
				destCol = '".$_POST['destCol']."',
				destRow = '".$_POST['destRow']."'
			WHERE 
				gameID = '".$_POST['gameID']."'";
		
			if ($conn->query($sql) === TRUE) 
			{
				$sql = "SELECT * FROM  `Games` WHERE  `gameID` = 1";
		        	// Execute SQL statement
		      		$result = mysqli_query($conn,$sql); 
		      		$row = $result->fetch_array(); 
		      		
		  		$gameStatusString = 
		  			"{
		  				status: 1,
		  				gameID : '".$_POST['gameID']."',
		  				numOfPlayers : '".$_POST['numOfPlayers']."',
		  				currentPlayerTurn: '".$_POST['currentPlayerTurn']."',
		  				sourceCol: '".$_POST['sourceCol']."',
		  				sourceRow: '".$_POST['sourceRow']."',
		  				destCol: '".$_POST['destCol']."',
		  				destRow: '".$_POST['destRow']."'
		  			 }";
		  			
				echo $gameStatusString;   
			} 
			else 
			{
			 
		      		$gameStatusString = 
	  				"{
		  				status: -1,
		  				gameID : 0,
		  				numOfPlayers : 0,
		  				currentPlayerTurn: 0,
		  				sourceCol: 0,
		  				sourceRow: 0,
		  				destCol: 0,
		  				destRow: 0
		  			 }";
				echo $gameStatusString;
			}
		
		}
		else{
			echo "ERROR: POST MUST include a gameID, numOfPlayers, currentPlayerTurn, sourceCol, sourceRow, destCol, and destRow field";
		}
	
	}
	//GET Request handler 
	elseif($_SERVER['REQUEST_METHOD'] === 'GET')
        {
        	if(isset($_GET['gameID']))
        	{
        	
	        	$sql = "SELECT * FROM  `Games` WHERE  `gameID` = '".$_GET['gameID']."'";
	        	// Execute SQL statement
	      		$result = mysqli_query($conn,$sql); 
	      		$row = $result->fetch_array();  
	  		
	  		//if invalid gameID
	  		if($row['gameID'] == "" )
	  		{
	  			$gameStatusString = 
  				"{
	  				status: -1,
	  				gameID : 0,
	  				numOfPlayers : 0,
	  				currentPlayerTurn: 0,
	  				sourceCol: 0,
	  				sourceRow: 0,
	  				destCol: 0,
	  				destRow: 0
	  			 }";
  			
			echo $gameStatusString;
	  		}
	  		else
	  		{
		  		$gameStatusString = 
	  			"{
	  				status: 1,
	  				gameID : '".$row['gameID']."',
	  				numOfPlayers : '".$row['numOfPlayers']."',
	  				currentPlayerTurn: '".$row['currentPlayerTurn']."',
	  				sourceCol: '".$row['sourceCol']."',
	  				sourceRow: '".$row['sourceRow']."',
	  				destCol: '".$row['destCol']."',
	  				destRow: '".$row['destRow']."'
	  			 }";
	  			
				echo $gameStatusString;   
			}
		} 
		else
		{
			$gameStatusString = 
  				"{
	  				status: -1,
	  				gameID : 0,
	  				numOfPlayers : 0,
	  				currentPlayerTurn: 0,
	  				sourceCol: 0,
	  				sourceRow: 0,
	  				destCol: 0,
	  				destRow: 0
	  			 }";
  			
			echo $gameStatusString;
		}       
        }

?>